api/Element.json

```diff
@@ -7932,7 +7933,10 @@
             },
             "firefox_android": "mirror",
             "oculus": "mirror",
-            "opera": "mirror",
+            "opera": {
+              "version_added": "64",
+              "notes": "Before version 126, `pointerrawupdate` events were exposed to non-secure contexts."
+            },
```

Comment:
> I also stopped the equivalent Opera data from mirroring, and gave it a separate version number and equivalent note. I did this because I didn't want the note to be exactly the same under the Opera data.

My expectation would be that `"mirror"` causes the mirrored note to have the Opera version corresponding to Chrome 142.

But maybe the problem is that there is no corresponding version yet, so I would hope this gets reverted to "mirror" as soon as we track Opera 126.